modules/core/src/errors.ts

```diff
@@ -132,10 +132,19 @@ export class AddressGenerationError extends BitGoJsError {
 }
 
 export class EthereumLibraryUnavailableError extends BitGoJsError {
+  private static readonly defaultMessage: string =
+    'Ethereum library required for operation is not available';
+
   public constructor(message?: string) {
-    super(message || 'Ethereum library required for operation is not available');
+    super(message || EthereumLibraryUnavailableError.defaultMessage);
```

Comment:
How about we allow the  message to just be the package name? Then we don't need `forImport`.

Are  there  any callers  using  the free-form message?

Reply:
I don't think so. Will change